CONTRIBUTING.md

```diff
@@ -1,64 +1,101 @@
-external submissions go through public reviews.
+Cobalt team members will be assigned to review your pull requests. Once the
+pull requests are approved and pass continuous integration checks, a Cobalt
+team member will merge the pull request.
 
+### Contribution guidelines and standards
 
-## Submission Process
+Before sending your pull request for
+[review](https://github.com/tensorflow/tensorflow/pulls),
+make sure your changes are consistent with the guidelines and follow the
+Cobalt coding style.
 
-We admit that this submission process is currently not completely optimized to
-make contributions easy, and we hope to make improvements to it in the
-future. It will always include some form of signing the CLA and submitting the
-code for review before merging changes into the Cobalt master tree.
+#### General guidelines and philosophy for contribution
 
-  1. Ensure you or your company have signed the appropriate CLA (see "Before You
-     Contribute" above).
-  1. Rebase your changes down into a single git commit.
-  1. Run `git clang-format HEAD~` to apply default C++ formatting rules,
-     followed by `git commit -a --amend` to squash any formatting changes
-     into your commit.
-  1. Run `git push origin HEAD:refs/for/master` to upload the review to
-     [Cobalt's Gerrit instance](https://cobalt-review.googlesource.com/).
-  1. Someone from the maintainers team will review the code, putting up comments
-     on any things that need to change for submission.
-  1. If you need to make changes, make them locally, test them, then `git commit
-     --amend` to add them to the *existing* commit. Then return to step 2.
-  1. If you do not need to make any more changes, a maintainer will integrate
-     the change into our private repository, and it will get pushed out to the
-     public repository after some time.
+*   Include unit tests when you contribute new features, as they help to a)
+    prove that your code works correctly, and b) guard against future breaking
+    changes to lower the maintenance cost.
+*   Bug fixes also generally require unit tests, because the presence of bugs
+    usually indicates insufficient test coverage.
+*   When you contribute a new feature to Cobalt, the maintenance burden is
+    (by default) transferred to the Cobalbt team. This means that the benefit
+    of the contribution must be compared against the cost of maintaining the
+    feature.
+*   As every PR requires several CPU/GPU hours of CI testing, we discourage
+    submitting PRs to fix one typo, one warning,etc. We recommend fixing the
+    same issue at the file level at least (e.g.: fix all typos in a file, fix
+    all compiler warning in a file, etc.)
+
+#### License
+
+Include a license at the top of new files. Check existing files for license examples.
+
+#### Coding style
+
+Changes to Cobalt C++ code should conform to
+[Google C++ Style Guide](https://google.github.io/styleguide/cppguide.html).
```

Comment:
Shouldn't this be Chrome/Chromium page ? https://chromium.googlesource.com/chromium/src/+/main/styleguide/styleguide.md

Reply:
done